src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,25 @@
+while (free W) {
+	move W
+}
+
+while (free N) {
+	move N
+}
+
+ while (free E) {
+	while (free S) {
+		move S
+	}
+
+	move E
+
+	while (free N) {
+		move N
+	}
+
+	move E
+}
+
+while (free S) {
+    move S
+}
```

Comment:
Tried to nest two while loops which didn't work (but seemed like you acknowledge this and this may have been white RightHand didn't work). Didn't get an error for this but Sophia and I also had been working on the project for so long and didn't get around to writing good error messages.